src/ResultsActivity/elements/PayOptions/PayOptions.js

```diff
@@ -47,35 +47,49 @@ function PayOptions({ offenderDetails, updateOffenderAttendance, date, openModal
     )
 
   const allowUpdate = () => {
-    if (selectedOption === 'other' && isWithinLastYear(date)) return true
+    if (selectedOption === 'other' && isWithinLastWeek(date)) return true
     return false
   }
+  const showRadioButton = () => {
+    if (!locked && offenderNo && eventId && isWithinLastWeek(date)) return true
+    return false
+  }
+
+  const payMsg = () => {
+    const msg =
+      isWithinLastWeek(date) && paid && locked ? 'Paid' : !isWithinLastWeek(date) && !paid && locked && 'Not Paid'
+    return msg
+  }
+
+  const otherMsg = () => {
+    const msg = (other || (!isWithinLastWeek(date) && !paid && !other)) && 'View/ Update'
```

Comment:
Don't need `!other` as you already know by the first condition.

Reply:
Spot.  Updated in latest push.